README.md

```diff
@@ -1,182 +1,13 @@
-
-
-## Prerequisites
-
-- [kind](https://kind.sigs.k8s.io/)
-- [kubectl](https://kubernetes.io/docs/reference/kubectl/)
-- [helm](https://helm.sh/)
-- [jsonnet](https://github.com/google/go-jsonnet)
-- [python 3.8+](https://www.python.org/) - needed for Monitoring of the KaaS layer use case
-- [make](https://www.gnu.org/software/make/)
-- [git](https://git-scm.com/)
-
-## Observer monitoring deployment
-
-### Create Kubernetes cluster
-
-```bash
-kind create cluster --config kind-observer-config.yaml --image kindest/node:v1.27.3 --name observer
-```
-
-If you opt not to use KinD and prefer utilizing an existing Kubernetes cluster,
-ensure that the metric endpoints for various control plane components are properly exposed.
-Refer to the [docs](https://dnationcloud.github.io/kubernetes-monitoring/helpers/FAQ/#kubernetes-monitoring-shows-or-0-state-for-some-control-plane-components-are-control-plane-components-working-correctly).
-
-### Deploy dnation-kubernetes-monitoring-stack
-
-Deploy dnation-kubernetes-monitoring-stack with SCS variables:
-
-_Optional_: Configure the object store as a long-term storage for metrics. Fill the
-`thanos-objstore.yaml` template manifest with the bucket credentials (refer to `thanosStorage.config`).
-
-_Optional_: Apply SCS brand secrets and optionally also letsencrypt issuer.
-  Uncomment also all SCS brand related parts in `values-observer.yaml`.
-  ```bash
-  kubectl apply -f scs/logo.yaml
-  kubectl apply -f scs/brand.yaml
-  kubectl apply -f scs/issuer.yaml  # Optional
-  ```
-
-```bash
-helm repo add dnationcloud https://dnationcloud.github.io/helm-hub/
-helm repo update dnationcloud
-helm upgrade --install dnation-kubernetes-monitoring-stack dnationcloud/dnation-kubernetes-monitoring-stack \
-  -f values-observer.yaml \
-  -f thanos-objstore.yaml  # Optional
-```
-
-### Optional: Monitoring of the KaaS layer deployment
-
-Note: This section is part of MVP0 version
-
-To test the Monitoring of the KaaS layer use case, deploy the Kaas-metric-importer
-into the Observer cluster.
-
-The Kaas-metric-importer is a simple service through which the KaaS software registers
-and unregisters newly created or deleted KaaS clusters in the Observer monitoring.
-This functionality enables the Observer monitoring to differentiate between KaaS clusters deleted intentionally
-and those that have stopped writing metrics to the Observer monitoring for any reason.
-
-```bash
-kubectl apply -f kaas-metric-importer.yaml
-```
-
-The Kaas-metric-importer uses an image built from https://github.com/m3dbx/prometheus_remote_client_golang.
-It has mounted configmap and based on configmap keys it pushes custom metric `kaas`
-with label `cluster` and value `1` into thanos receiver.
-E.g. configmap:
-```yaml
-data:
-  workload-cluster: ""
-```
-It pushes metric e.g. `kaas{cluster="workload-cluster"} 1` to the Observer.
-
-
-### Optional: Monitoring of infrastructure services deployment
```

Comment:
Where is the documentation about this option?

Reply:
thank you for pointing this out; here it is: 51f377fd2350183356dd8e625b8c0a83ebfd9b7e